tests/utils.toit

```diff
@@ -655,7 +655,24 @@ class TestDevicePipe extends TestDevice:
         --alias-id=alias-id
         --organization-id=organization-id
 
-    fork_ "bash" [boot-sh]
+    fork-bash_ boot-sh
+
+  fork-bash_ script-path/string -> none:
+    bash := "bash"
+    if system.platform == system.PLATFORM-WINDOWS:
+      // Running on Windows is tricky...
+      // - We want Git's bash and not any other. The path to the git-bash must
+      //   be in Windows format.
+      // - We provide a shell script as argument. That script must be in Unix
+      //   (cygwin) format.
+      // Example: `C:/Program Files/Git/usr/bin/bash.exe /tmp/envelope-test-...`.
+      script-path = (pipe.backticks ["cygpath", "-u", script-path]).trim
+      program-files-path := os.env.get "ProgramFiles"
+      if not program-files-path:
+        // This is brittle, as Windows localizes the name of the folder.
```

Comment:
Maybe extend the comment to explain why it is good enough for our testing?

Reply:
Removed it again.
We can't nicely kill the subprocesses on Windows, so I just skip them.